README.md

```diff
@@ -149,6 +149,9 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
 |changelog\_project|Sets the github project name for the change\_log\_generator rake task. Optional, if not set it will parse the `source` from the `metadata.json` file|
 |changelog\_since\_tag|Sets the github since_tag for the change\_log\_generator rake task. Required for the changlog rake task.|
 |changelog\_version\_tag\_pattern|Template how the version tag is to be generated. Defaults to `'v%s'` which eventually align with tag\_pattern property of puppet-blacksmith, thus changelog is referring to the correct version tags and compare URLs. |
+|github_site|Override built-in default for public GitHub. Useful for GitHub Enterprise and other. |
```

Comment:
Do you have any examples to add here. For example, would it be `https://xxx.internal.domain` or just `xxx.internal.domain`.  Or a reference to where I'd get more information about what values are valid here.

Reply:
Also for the other settings below.